File: safe_place/v1/get/safety_rated_locations/NodeSDK/safe_place.js
var Amadeus = require("amadeus");
var amadeus = new Amadeus({
  clientId: 'YOUR_API_KEY',
  clientSecret: 'YOUR_API_SECRET'
});

// How safe is Barcelona? (based a geo location and a radius)
amadeus.safety.safetyRatedLocations.get({
  latitude: 41.397158,
  longitude: 2.160873
}).then(function (response) {
  console.log(response);
}).catch(function (response) {
  console.error(response);
});

File: safe_place/v1/get/safety_rated_locations/iOS SDK/SafePlace.swift
import Amadeus

var amadeus: Amadeus = Amadeus(
    client_id: "YOUR_API_ID",
    client_secret: "YOUR_API_SECRET"
)

// How safe is Barcelona? (based a geo location and a radius)
amadeus.safety.safetyRatedLocations.get(params: ["latitude": "41.397158",
                                                 "longitude": "2.160873",
                                                 "radius": "2"],
                                                     onCompletion: {
                                                            result in
                                                            switch result {
                                                            case .success(let response):
                                                                print(response.data)
                                                            case .failure(let error):
                                                                print(error.localizedDescription)
                                                            }
                                        })
